.github/workflows/integration.yml

```diff
@@ -57,7 +57,7 @@ jobs:
       # Prerequisites
       - uses: ./.github/actions/restore-node
         with:
-          node-version: 16.x
+          node-version: 18.18
```

Comment:
It looks like the getting started test still clones agoric-sdk. That's no longer part of the workflow.

Reply:
_I think_ we still need to clone agoric-sdk, because with `yarn create @agoric/dapp ...` we're implicitly testing the behavior of `init` subcommand of agoric-cli package: https://github.com/Agoric/agoric-sdk/blob/master/packages/agoric-cli/src/main.js#L124-L140

Reply:
Manual testing has confirmed that we don't need to clone agoric-sdk. `yarn create @agoric/dapp` gets code to run `agoric init` from npm, not from an agoric-sdk clone. I sure hope we can do likewise in ci.

This ci job says something about a local registry. That's perhaps the way `yarn create @agoric/dapp ...` gets access to the code under test.

Reply:
> This ci job says something about a local registry. That's perhaps the way yarn create @agoric/dapp ... gets access to the code under test.

Yeah I think we [start a local NPM registry](https://github.com/Agoric/agoric-sdk/blob/master/.github/workflows/integration.yml#L84-L90) before running getting-started tests and we publish via a CI specific tag [here](https://github.com/Agoric/agoric-sdk/blob/master/scripts/registry.sh#L47).

...actually I'm not so sure anymore, just pushed a debug commit to weed out what's using local code and what's using npmjs.com

Reply:
OK I just confirmed that `yarn create @agoric/dapp ...` will always pull from npmjs.com and I was not able to find a way to make it pull from local registry with my very limited yarn knowledge. I also confirmed that `agoric init ...` will use the local registry and code.

Here's what I propose:
1. in agoric-sdk repo, we use `agoric init ...` to initialize `dapp-offer-up` (this PR)
    1. this way, we ensure integration tests in agoric-sdk repo are testing the behavior of `agoric init ...` at git HEAD, for example, if I change `DEFAULT_DAPP_TEMPLATE` to something erroneous, [integration tests will fail right away](https://github.com/Agoric/agoric-sdk/actions/runs/8254602776/job/22579135135)
    2. merging this PR would also unblock @turadg's PRs to deprecate support for node 16, add support for node 20, and migrate from yarn classic to yarn modern
2. in dapp-offer-up repo, we use `yarn create @agoric/dapp ...` to initialize `dapp-offer-up` followed by integration tests
    1. this way, we ensure integration tests in dapp-offer-up repo are testing the behavior of `agoric init ...` that's published on npmjs.com
    2. [this work](https://github.com/Agoric/dapp-offer-up/issues/48) is not scheduled, and IMO is not high priority
    
@dckc sounds like a plan?

Reply:
yes. sounds good.